static/js/place/main.tsx

```diff
@@ -96,13 +97,18 @@ class MainPane extends React.Component<MainPanePropType> {
                 <a href={`/place/${this.props.dcid}?topic=${topic}`}>{topic}</a>
                 <span className="more">
                   <a href={`/place/${this.props.dcid}?topic=${topic}`}>
-                    More charts ›
+                    <Trans i18nKey="more_charts">More charts ›</Trans>
                   </a>
                 </span>
               </h3>
             );
           } else {
-            subtopicHeader = <h3 id={topic}>{topic}</h3>;
+            // XXX find a way to translate these.
```

Comment:
I think maybe we don't need to have the topic be auto extracted since they are in the chart config? We can write our extractor for chart config

Reply:
I was testing if the topic would get translated (hence the manually crafted es.json file)